pca_reduction.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+import numpy as np
+#import sys
+#sys.path.insert(1,r"C:\Users\Yakoubi\regression_pipeline")
+from data_preparation import *
+from sklearn.decomposition import PCA
+
+
+def reduce_dimensionality(dataset: pd.DataFrame):
+    # Function to load dataset and apply principal component analysis in order to reduce dimensionality. Return the dimension reduced features and the labels in one dataframe.
```

Comment:
Documenting functions is done this way : https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format, using the `reST` format.